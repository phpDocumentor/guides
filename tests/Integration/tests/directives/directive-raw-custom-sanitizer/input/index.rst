Directive tests
===============

.. raw:: html

   <div class="someClass">
     <script>alert('XSS');</script>
     <p>Lorem Ipsum Dolor</p>
     <h2>Some Rubric</h2>
     <p>Dolor sit!</p>
   </div>
