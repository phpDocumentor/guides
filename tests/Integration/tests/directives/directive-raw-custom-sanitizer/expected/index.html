<!-- content start -->
    <div class="section" id="directive-tests">
            <h1>Directive tests</h1>


  <p>Lorem Ipsum Dolor</p>

  <p>Dolor sit!</p>

    </div>
<!-- content end -->
